Sources/Models/Views/StackView.swift

```diff
@@ -37,6 +35,20 @@ public struct StackView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let backgroundColor: Color?
     public let tintColor: Color?
 
+    public let axis: String
+    public let distribution: String?
+    public let alignment: String?
+    public let spacing: Int
+    public let baselineRelativeArrangement: Bool
```

Comment:
I think they should be optional if they are not required fields.

Reply:
ok no default value ✅